src/resources/Pipelines/ResultRankings/ResultRankingsInterfaces.ts

```diff
@@ -0,0 +1,90 @@
+import {ListStatementSortBy} from '../Statements';
+
+export enum ResultRankingsKind {
+    featuredResults = 'featured_result',
+    rankingExpression = 'ranking_expression',
+}
```

Comment:
This enum will need to be moved inside [Enum.ts](https://github.com/coveo/platform-client/blob/0812c70ee5da3d1d1adc7737e621e158cc51f70a/src/resources/Enums.ts) otherwise it won't be included in the resulting js file